frontend/ui/widget/virtualkeyboard.lua

```diff
@@ -586,7 +595,7 @@ function VirtualKeyPopup:init()
     local keyboard_frame = FrameContainer:new{
         margin = 0,
         bordersize = Size.border.default,
-        background = Blitbuffer.COLOR_WHITE,
+        background = G_reader_settings:readSetting("night_mode") and Blitbuffer.COLOR_WHITE or Blitbuffer.COLOR_LIGHT_GRAY,
```

Comment:
Same as below (I may be reading this bottom to top, sorry, caught my eye while scrolling ;)).